java/daikon/PptTopLevel.java

```diff
@@ -4405,6 +4405,19 @@ public boolean is_exit() {
     }
   }
 
+  // UNDONE: this doesn't look right
```

Comment:
Does "UNDONE" mean  "TODO"?  I think the "UNDONE" comments should be clarified or removed, throughout.